docking.js

```diff
@@ -184,7 +184,7 @@ var DockedDash = GObject.registerClass({
     }
 }, class DashToDock extends St.Bin {
 
-    _init(monitorIndex) {
+    _init(monitorIndex, isMain) {
```

Comment:
sounds good, i just wasn't aware of this option since i'm in no way familiar with gobject internals, this is the first time i'm involved with it :) 

for some reason this change isn't listed as a "suggested change" so i can't commit it with one click. can you maybe change that?

Reply:
Sadly I can't as I can't suggest a such big change... 

Just copy and paste it to a new file and apply it with `git apply file.patch` or `patch -p1 < file.patch`

Reply:
implemented and squashed (patch didn't work so i manually edited it, please check if it's okay)